docs/release-notes/NuGet-6.3.md

```diff
@@ -0,0 +1,125 @@
+---
+title: NuGet 6.3 Release Notes
+description: Release notes for NuGet 6.3 including new features, bug fixes, and DCRs.
+author: <GithubAlias>
+ms.author: <MicrosoftAlias>
+ms.date: 8/2/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.3 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**<NuGetVersion>**](https://nuget.org/downloads) | [Visual Studio <VSYear> version <VSVersion>](https://visualstudio.microsoft.com/downloads/) | [<SDKVersion>](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio <VSYear> with.NET Core workload
+
+## Summary: What's New in 6.3
+
+* Implement "consuming pdbs from packages in PackageReference" - [#11352](https://github.com/NuGet/Home/issues/11352)
+
+* Create experimentation metrics to measure engagement of transitive dependencies - [#10759](https://github.com/NuGet/Home/issues/10759)
+
+* Enhance the experimentation infrastructure in NuGet code to support transitive dependencies - [#10758](https://github.com/NuGet/Home/issues/10758)
+
+* Allow to user to input custom (floating) versions through the PM UI - [#9829](https://github.com/NuGet/Home/issues/9829)
+
+* [Feature] Error on multiple entries for the same package ids in Directory.Packages.props - [#9467](https://github.com/NuGet/Home/issues/9467)
```

Comment:
I think we should combine this with tre rest of the duplicate id reporting. 

Duplicate PackageReference includes are handled inconsistently by all restores, leading to unnecessary full restores.

Should we maybe combine them into a smarter title and link both issues?

Maybe we say 

` NuGet warns when duplicate PackageReference, PackageVersion or PackageDownload items are specified`

Reply:
Can we get a quick blurb of this error code in addition of the blurb you propose at the bottom? I can clean it up later!

Reply:
What else did you have in mind? 

NU1504, NU1505 and NU1506 are the warning codes and they are all already documented. https://github.com/NuGet/docs.microsoft.com-nuget/blob/main/docs/reference/errors-and-warnings/NU1504.md

Reply:
@nkolev92 Just trying to collect all the literature here. NU1504-1506 is what I'll need to link to :)